src/c_kzg_4844.c

```diff
@@ -33,6 +33,14 @@
 #define CHECK(cond) \
     if (!(cond)) return C_KZG_BADARGS
 
+/** Helper macro to release memory allocated on the heap.  Unlike free(),
+ * c_kzg_free() macro sets the pointer value to NULL after freeing it. */
```

Comment:
For consistency, please:

* start the comment on the second line after the `/**`,
* end the block comment with `*/` on a new line,
* and use a single space between sentences.

```suggestion
/**
 * Helper macro to release memory allocated on the heap. Unlike free(),
 * c_kzg_free() macro sets the pointer value to NULL after freeing it.
 */
```